pdns/recpacketcache.cc

```diff
@@ -195,15 +191,15 @@ void RecursorPacketCache::insertResponsePacket(unsigned int tag, uint32_t qhash,
     }
 
     d_packetCache.insert(e);
-  }
-}
 
-uint64_t RecursorPacketCache::size()
-{
-  return d_packetCache.size();
+    if (d_packetCache.size() > d_maxSize) {
+      auto it = d_packetCache.begin();
```

Comment:
I would expect this to take the first item of the first index of `d_packetCache`, so the `HashTag` one, and not the ` SequencedTag`? In any case I would prefer explicitly using the `SequencedTag` index so our intention is clear.

Reply:
Ugh, thanks for spotting that!